cli/src/commands/run_config.py

```diff
@@ -17,5 +17,5 @@ def get_options(self):
     def run(self, args):
         from scraper.src.index import run_config
 
-        self.check_not_docsearch_app_id("run a config manually")
-        return run_config(args[0])
+        # self.check_not_docsearch_app_id("run a config manually")
+        return run_config(args[0])
```

Comment:
Keep the new line at the end, for homogeneity

Reply:
Removed, wrong git add